cinema/serializers.py

```diff
@@ -1 +1,97 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import (CinemaHall,
+                           Genre,
+                           Actor,
+                           Movie,
+                           MovieSession,
+                           Order,
+                           Ticket)
```

Comment:
```suggestion
from cinema.models import (
    CinemaHall,
    Genre,
    Actor,
    Movie,
    MovieSession,
    Order,
    Ticket
)
```
please fix in other places